daffodil-japi/src/main/java/org/apache/daffodil/japi/XMLOutputStyle.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.daffodil.japi;
+
+/**
+ * XMLOutputStyles for determining whether to wrap info in CDATA chunks
+ */
+public enum XMLOutputStyle {
+  /**
+   * No CDATA Wrapping
```

Comment:
We should probably describe what this does do instead of what it doesn't do. In this case, infoset strings are written as is, and special characters (e.g. < > & ", maybe others?) are escaped using XML &-escaping. Where as PrettyPrintSafe does not do that escaping, but instead wraps infoset strings in CDATA tags.